files/scripts/swss.sh

```diff
@@ -126,6 +126,12 @@ function clean_up_chassis_db_tables()
 {
 
     switch_type=`$SONIC_DB_CLI CONFIG_DB  hget 'DEVICE_METADATA|localhost' 'switch_type'`
+    platform=`$SONIC_DB_CLI CONFIG_DB  hget 'DEVICE_METADATA|localhost' 'platform'`
+    chassis_config="/usr/share/sonic/device/$platform/chassisdb.conf"
+    if [ ! -e $chassis_config ]; then
+       debug "No chassis config found"
+       return
+    fi
```

Comment:
This new section of code should be moved after line 139 which is the end of the line 137 (if condition block)